source/includes/steps-fle-configure-the-mongodb-client.yaml

```diff
@@ -87,7 +87,7 @@ content: |
            :emphasize-lines: 2
 
            HashMap<String, BsonDocument> schemaMap = new HashMap<String, BsonDocument>() {{
-             put("medicalRecords.patients", BsonDocument.parse(jsonSchema));
+               put("medicalRecords.patients", BsonDocument.parse(jsonSchema));
```

Comment:
Same here: avoid the anonymous class creation

Reply:
👍 Got it, wasn't sure if you were only referring to the examples you previously identified or in general. Will update.